content/index/index.html

```diff
@@ -39,430 +39,430 @@ <h2 id="examples_by_role_label">Examples by Role</h2>
+                <li><a href="../patterns/dialog-modal/examples/datepicker-dialog.html">日付ピッカーダイアログの例</a> (<abbr title="ハイコントラストサポート">HC</abbr>)</li>
+                <li><a href="../patterns/dialog-modal/examples/dialog.html">モーダルダイアログの例</a></li>
               </ul>
             </td>
           </tr>
           <tr>
             <td><code>feed</code></td>
-            <td><a href="../patterns/feed/examples/feed.html">Infinite Scrolling Feed</a></td>
+            <td><a href="../patterns/feed/examples/feed.html">無限スクロールフィードの例</a></td>
           </tr>
           <tr>
             <td><code>form</code></td>
-            <td><a href="../patterns/landmarks/examples/form.html">Form Landmark</a></td>
+            <td><a href="../patterns/landmarks/examples/form.html">フォームランドマークの例</a></td>
           </tr>
           <tr>
             <td><code>grid</code></td>
             <td>
               <ul>
-                <li><a href="../patterns/combobox/examples/combobox-datepicker.html">Date Picker Combobox</a> (<abbr title="High Contrast Support">HC</abbr>)</li>
-                <li><a href="../patterns/combobox/examples/grid-combo.html">Editable Combobox with Grid Popup</a></li>
-                <li><a href="../patterns/dialog-modal/examples/datepicker-dialog.html">Date Picker Dialog</a> (<abbr title="High Contrast Support">HC</abbr>)</li>
-                <li><a href="../patterns/grid/examples/data-grids.html">Data Grid</a></li>
-                <li><a href="../patterns/grid/examples/layout-grids.html">Layout Grid</a></li>
+                <li><a href="../patterns/combobox/examples/combobox-datepicker.html">日付ピッカーコンボボックスの例</a> (<abbr title="ハイコントラストサポート">HC</abbr>)</li>
+                <li><a href="../patterns/combobox/examples/grid-combo.html">グリッドポップアップ付き編集可能コンボボックスの例</a></li>
+                <li><a href="../patterns/dialog-modal/examples/datepicker-dialog.html">日付ピッカーダイアログの例</a> (<abbr title="ハイコントラストサポート">HC</abbr>)</li>
+                <li><a href="../patterns/grid/examples/data-grids.html">データグリッドの例</a></li>
+                <li><a href="../patterns/grid/examples/layout-grids.html">レイアウトグリッドの例</a></li>
               </ul>
             </td>
           </tr>
           <tr>
             <td><code>gridcell</code></td>
             <td>
               <ul>
-                <li><a href="../patterns/combobox/examples/grid-combo.html">Editable Combobox with Grid Popup</a></li>
-                <li><a href="../patterns/grid/examples/layout-grids.html">Layout Grid</a></li>
-                <li><a href="../patterns/treegrid/examples/treegrid-1.html">Treegrid Email Inbox</a></li>
+                <li><a href="../patterns/combobox/examples/grid-combo.html">グリッドポップアップ付き編集可能コンボボックスの例</a></li>
+                <li><a href="../patterns/grid/examples/layout-grids.html">レイアウトグリッドの例</a></li>
+                <li><a href="../patterns/treegrid/examples/treegrid-1.html">ツリーグリッドメールインボックスの例</a></li>
```

Comment:
「インボックス」はそこまでメジャーな言い回しではない気がしました。
一案ですが…
```suggestion
                <li><a href="../patterns/treegrid/examples/treegrid-1.html">ツリーグリッドによるメール受信トレイの例</a></li>
```
こんな感じはいかがでしょうか？

Reply:
Gmailでも Inbox / 受信トレイ という訳を採用していたので、いただきました :pray:
fcbbb5da3af575cf3bfddfacfa5b3ad85da9a7ed